packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -4466,7 +4498,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
           const xFilterBar = gridX + columnX + columnWidth - 20;
           const style: CSSProperties = isFilterBarShown
             ? {
-                position: 'absolute',
+                position: 'ßabsolute',
```

Comment:
```suggestion
                position: 'absolute',
```

Reply:
Yeah saw this, have already made change in local branch, not sure how it got there, I do not even know how to get that letter from my keyboard